pages/linux/esearch.md

```diff
@@ -8,6 +8,14 @@
 
 `esearch -db pubmed -query "{{selective serotonin reuptake inhibitor}}"`
 
+- Search the protein database using a query and regexp:
+
+`esearch -db {{protein}} -query {{'Escherichia*'}}`
+
 - Search the nucleotide database for sequences whose metadata contain insulin and rodents:
 
 `esearch -db nuccore -query "{{insulin [PROT] AND rodents [ORGN]}}"`
+
+- Display help:
```

Comment:
```suggestion
- Display [h]elp:
```

Reply:
Are you sure? It isn't invalid, but seems a really simple example to highlights the argument in the description

Reply:
the `-h` or `-help` option displays all command line options for esearch

Reply:
> Are you sure? It isn't invalid, but seems a really simple example to highlights the argument in the description

It is valid. I usually don't make this suggestion for help command. But I found this in the docs `Each program has a ‑help command that prints detailed information about available arguments.` so I made the suggestion.